<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="AgBe" failures="2" tests="16" name="com.tvs.pgm.twoW.mobile.tests.DashboardTest" time="28.055" errors="0" timestamp="2025-07-21T21:03:03 IST" skipped="0">
  <testcase classname="com.tvs.pgm.twoW.mobile.tests.DashboardTest" name="verifyHelpText" time="0.107"/>
  <testcase classname="com.tvs.pgm.twoW.mobile.tests.DashboardTest" name="clickOnHelp" time="1.452"/>
  <testcase classname="com.tvs.pgm.twoW.mobile.tests.DashboardTest" name="verifyCreateOrRequestQueryText" time="0.306"/>
  <testcase classname="com.tvs.pgm.twoW.mobile.tests.DashboardTest" name="clickCreateOrRequestQuery" time="1.113"/>
  <testcase classname="com.tvs.pgm.twoW.mobile.tests.DashboardTest" name="verifyQueryTitleText" time="0.171">
    <failure message="expected [Enter Here] but found [null]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Enter Here] but found [null]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at com.tvs.pgm.twoW.mobile.tests.DashboardTest.verifyQueryTitleText(DashboardTest.java:208)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- verifyQueryTitleText -->
  <testcase classname="com.tvs.pgm.twoW.mobile.tests.DashboardTest" name="clickAndEnterQueryTitle" time="1.380"/>
  <testcase classname="com.tvs.pgm.twoW.mobile.tests.DashboardTest" name="verifySelectCategoryText" time="0.769"/>
  <testcase classname="com.tvs.pgm.twoW.mobile.tests.DashboardTest" name="clickOnSelectCategory" time="0.258"/>
  <testcase classname="com.tvs.pgm.twoW.mobile.tests.DashboardTest" name="verifyChooseCategoryText" time="0.946"/>
  <testcase classname="com.tvs.pgm.twoW.mobile.tests.DashboardTest" name="clickChooseCategory" time="1.094"/>
  <testcase classname="com.tvs.pgm.twoW.mobile.tests.DashboardTest" name="verifyAddDescriptionText" time="0.150">
    <failure message="expected [Enter Description Here] but found [null]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Enter Description Here] but found [null]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at com.tvs.pgm.twoW.mobile.tests.DashboardTest.verifyAddDescriptionText(DashboardTest.java:241)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- verifyAddDescriptionText -->
  <testcase classname="com.tvs.pgm.twoW.mobile.tests.DashboardTest" name="clickAndEnterAddDescription" time="3.087"/>
  <testcase classname="com.tvs.pgm.twoW.mobile.tests.DashboardTest" name="verifyCreateQueryText" time="0.706"/>
  <testcase classname="com.tvs.pgm.twoW.mobile.tests.DashboardTest" name="clickOnCreateQuery" time="2.251"/>
  <testcase classname="com.tvs.pgm.twoW.mobile.tests.DashboardTest" name="clickOnHelpQuery" time="0.674"/>
  <testcase classname="com.tvs.pgm.twoW.mobile.tests.DashboardTest" name="verifyAllQueryStatus" time="13.591"/>
</testsuite> <!-- com.tvs.pgm.twoW.mobile.tests.DashboardTest -->
